### DISCUSSION TITLE
Does/Will mini.completion support LSP function signature snippets?

### DISCUSSION THREAD

**Discussion by ldelossa** [nvim-mini/mini.nvim#1380] [Q&A] [answered]
Hello.

A lot of LSPs will include snippets when a function is completed.
Neovim has a built in method for handling these but it requires you also use its built in LSP completion feature as well.

Does or will mini.completion support snippet expansion? What I mean by this is when a function is being completed hitting <Tab> will move you to each argument where you can fill in the variables you want to include in the final function call syntax.

**Comment by echasnovski** [comment, marked as answer]
Yes, snippet support in 'mini.completion' is planned. See #886.
